src/app/components/treeselect/treeselect.css

```diff
@@ -94,4 +94,12 @@
     .p-treeselect-clearable {
         position: relative;
     }
+
+    .p-treeselect-virtual-scroll .p-tree.p-component {
+      padding: 0;
+    }
+
+    .p-treeselect-virtual-scroll .p-tree-container.p-scroller-content {
+      padding: 1.25rem;
```

Comment:
If I'm not wrong this styles themes should be in `primeng-sass-theme` project: https://github.com/primefaces/primeng-sass-theme/
Maybe @cetincakiroglu can confirm you

Reply:
Can i have a confirmation ?